Packs/CortexAttackSurfaceManagement/IncidentFields/incidentfield-ASM_-_Service_Owner.json

```diff
@@ -65,7 +65,7 @@
             "fieldCalcScript": "",
             "isDefault": true,
             "isReadOnly": false,
-            "key": "rankingscore",
+            "key": "ranking_score",
```

Comment:
Where this field is being used? Moreover, how can we ensure that this change won't affect current users who are mapping to this field?

Reply:
Current users that are mapping to this field should not be getting back any data. This field is **only** set by our 
 [RankServiceOwners](https://github.com/demisto/content/tree/master/Packs/CortexAttackSurfaceManagement/Scripts/RankServiceOwners) script and more buy the backend on the tenant.

Reply:
Also this in an ASM field and ASM PR that is currently only being populated and used by Xpanse customers in Xpanse tenants, not XSOAR at this time.

Reply:
Got ya. Cool by me! 🎉